apps/antalmanac/src/components/Calendar/CalendarToolbar.tsx

```diff
@@ -38,17 +40,35 @@ const styles: Styles<Theme, object> = {
         flexGrow: 1,
     },
     scheduleSelector: {
-        marginLeft: '10px',
-        maxWidth: '9rem',
+        paddingLeft: '1em',
+        width: '10rem',
+    },
+    termSelector: {
+        flexGrow: 1,
+        flexShrink: 1,
+        display: 'flex',
+        overflow: 'hidden',
+        paddingLeft: '1rem',
+        paddingRight: '1rem',
```

Comment:
There should be a padding inline CSS variable which does padding left and right https://developer.mozilla.org/en-US/docs/Web/CSS/padding-inline

In MUIv5, you can also use `px` or `paddingX`